### PR TITLE
FINERACT-1345: Fixing support for parameterType reports

### PR DESCRIPTION
**Pull request by ptuomola**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [x] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [x] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadReportingServiceImpl.java [hunk 1]
```diff
@@ -212,7 +212,9 @@ private String getSql(final String name, final String type) {
     }
 
     @Override
-    public String getReportType(final String reportName, final boolean isSelfServiceUserReport) {
+    public String getReportType(final String reportName, final boolean isSelfServiceUserReport, final boolean isParameterType) {
+        if (isParameterType) return "Table";
```
Comment (josemakara2) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadReportingServiceImpl.java]: Format violations
`if` construct must use {}'s.

**Review by josemakara2 — CHANGES_REQUESTED** [review]
Minor change required there. See comment on `ReadReportingServiceImpl.java`

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadReportingServiceImpl.java (hunk 1 above)
Comment (ptuomola) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadReportingServiceImpl.java]: Thanks - have fixed... strange that spotlessCheck did not complain, we obviously have two different sets of formatting rules in place. Given you've confirmed this works for you, I'll merge this if it passes the automatic tests.

**Review by josemakara2 — APPROVED** [review]
LGTM

**Pull request merged**